tests/lib/nested.sh

```diff
@@ -232,9 +232,32 @@ create_nested_core_vm(){
 
             snap download --basename=pc-kernel --channel="20/edge" pc-kernel
             uc20_build_initramfs_kernel_snap "$PWD/pc-kernel.snap" "$WORK_DIR/image"
-            EXTRA_FUNDAMENTAL="--snap $WORK_DIR/image/pc-kernel_*.snap"
-            chmod 0600 "$WORK_DIR"/image/pc-kernel_*.snap
+
+            # Prepare the pc kernel snap
+            KERNEL_SNAP=$(ls "$WORK_DIR"/image/pc-kernel_*.snap)
```

Comment:
why are we not also re-packing the kernel.efi initramfs with snap-bootstrap like we do for normal UC20 prepare script? i.e. this whole snippet: https://github.com/snapcore/snapd/blob/master/tests/lib/prepare.sh#L432 ? 

Is this only run nightly?

Reply:
It is just for nightly execution and this test suite does not make the same preparation then the regular main suite.

Initially was running the prepare suite but then it was removed to make it faster because in the prepare we do many thinks that are not needed for nested tests, mostrly on uc16 and uc18

Reply:
Okay sounds good to me then